packages/react-jss/src/withStyles.js

```diff
@@ -39,6 +39,29 @@ const getStyles = <Theme: {}>(styles: Styles<Theme>, theme: Theme) => {
   if (typeof styles !== 'function') {
     return styles
   }
+  if (process.env.NODE_ENV !== 'production') {
+    if (styles.length < 1) {
+      console.warn(
+        `
+        Warning: [JSS]: This Component uses themed styles notation (function) while not relying on a theme (0 arguments).
+        It slows your app down and you should rewrite these styles to plain object notation.
```

Comment:
I would remove the slows down part, because one might interpret it as something that is really really slow, instead of "it could be a bit better, because styles are removed across the element". I think we can skip the explanation. Just give a hint what to do.

Reply:
I disagree.

> it could be a bit better

This part will give anyone free pass to not do any optimisations and people will complain to remove the warning.

> styles are removed across the element

I am not sure I understand what does it mean.